FirebaseVertexAI/Tests/Integration/IntegrationTests.swift

```diff
@@ -13,9 +13,10 @@
 // limitations under the License.
 
 import FirebaseCore
-import FirebaseVertexAI
 import XCTest
 
+@testable import FirebaseVertexAI
```

Comment:
Why testable? What's now an internal API?

Reply:
`FunctionCall` doesn't have a public constructor since it's typically instantiated with `init(from decoder:)`.
https://github.com/firebase/firebase-ios-sdk/blob/688af8041a6515aa778197ba71f1e6f2a03d711e/FirebaseVertexAI/Sources/FunctionCalling.swift#L17-L24

Looks like the other platforms have this as public though so I think I'll add it instead of `@testable`.

Reply:
Done. Thanks for calling this out. I could see it being useful for 3P devs making SwiftUI previews for function calling features as well.